conda/core/initialize.py

```diff
@@ -1081,27 +1081,51 @@ def _bashrc_content(conda_prefix, shell):
         }
     else:
         conda_exe = join(conda_prefix, 'bin', 'conda')
-        conda_initialize_content = dals("""
-        # >>> conda initialize >>>
-        # !! Contents within this block are managed by 'conda init' !!
-        __conda_setup="$('%(conda_exe)s' 'shell.%(shell)s' 'hook' 2> /dev/null)"
-        if [ $? -eq 0 ]; then
-            eval "$__conda_setup"
-        else
-            if [ -f "%(conda_prefix)s/etc/profile.d/conda.sh" ]; then
-                . "%(conda_prefix)s/etc/profile.d/conda.sh"
+
+        if shell in ("csh", "tcsh"):
+            conda_initialize_content = dals("""
+            # >>> conda initialize >>>
+            # !! Contents within this block are managed by 'conda init' !!
+            set __conda_setup="`%(conda_exe)s shell.%(shell)s hook >& /dev/null`"
+            if ( $? == 0 ) then
+                eval "$__conda_setup"
+            else
+                if ( -f "%(conda_prefix)s/etc/profile.d/conda.sh" ) then
+                    . "%(conda_prefix)s/etc/profile.d/conda.sh"
+                else
+                    export PATH="%(conda_bin)s:$PATH"
```

Comment:
This would also fail because `export` is not csh, although I am not sure whether, although I am not sure whether this 
 `else` branch is ever executed.

Reply:
ohh, good catch. Should this be `setenv` instead? 
I think this bit will pretty much never get executed. I believe it's there in case users downgrade conda and the newer activation bits are removed.